client/verta/tests/constants.py

```diff
@@ -28,6 +28,8 @@
 EMAIL_SYS_ADMIN = os.environ.get(EMAIL_SYS_ADMIN_ENV_VAR)
 DEV_KEY_SYS_ADMIN = os.environ.get(DEV_KEY_SYS_ADMIN_ENV_VAR)
 
+NAMESPACE = "cluster1--namespace1"
```

Comment:
```suggestion
MNS_NAMESPACE = "cluster1--namespace1"
```

or something, to suggest what it's purpose is